flowredux/src/commonMain/kotlin/com/freeletics/flowredux/dsl/FlowReduxStateMachine.kt

```diff
@@ -40,12 +34,7 @@ public abstract class FlowReduxStateMachine<S : Any, A : Any>(
 
         outputState = inputActions
             .receiveAsFlow()
-            .map<A, Action<S, A>> { ExternalWrappedAction(it) }
-            .onStart {
-                emit(InitialStateAction())
-            }
-            .reduxStore(initialStateSupplier, sideEffects, ::reducer)
-            .distinctUntilChanged { old, new -> old === new } // distinct until not the same object reference.
```

Comment:
dont we need the distinct until changed?

Reply:
We have this check inside the store now https://github.com/freeletics/FlowRedux/pull/501/files#diff-0bd32fee6f00b61666521e8846b321dce06c0dbf4ea5266f85bc6a7abd741e6bR42 It has the advantage that we also don't run the side effects again if the state stayed identical.